desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -922,6 +925,43 @@
    # Django cache to use to store temporarily used data during query execution. This is in addition to result_file_storage and result_backend.
    ## execution_storage='{"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "celery-hue"}'
 
+  # Configuration options for the Task Server V2
+  # ------------------------------------------------------------------------
+  [[task_server_v2]]
+
+   # If resource intensive or blocking can be delegated to an already running task server.
+   ## enabled=False
+
+   # Switch on the integration with the Task Scheduler.
+   ## beat_enabled=False
+
+   # Number of query results rows to fetch into the result storage.
+   ## fetch_result_limit=2000
+
+   # Django file storage class to use to temporarily store query results
+   ## result_storage='{"backend": "django.core.files.storage.FileSystemStorage", "properties": {"location": "./logs"}}'
+
+   # How the task server and tasks communicate.
+   ## broker_url=amqp://guest:guest@localhost//
+
+   # Where to store task results. Defaults to local file system path. Celery comes with a several other backends.
+   ## celery_result_backend=file:///$HUE_ROOT/logs
+
+   # Default options provided to the task server at startup.
+   ## celeryd_opts='--time-limit=300'
+
+   # Django cache to use to store temporarily used data during query execution. This is in addition to result_file_storage and result_backend.
+   ## execution_storage='{"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "celery-hue"}'
+
+   # Time interval in seconds to run this peroidic cleaner which checks disk usage and makes space for file uploads
+   ## check_disk_usage_and_clean_task_periodic_interval=1000
+
+   # Clean up files in /tmp folder if the disk usage is beyond the threshold
+   ## disk_usage_cleanup_threshold=90
+
+   # Time interval in seconds to run this peroidic cleaner which cleans up failed upload tasks stored in redis
+   ## cleanup_stale_uploads_task_periodic_interval
```

Comment:
please keep conf.dist/hue.ini and conf/pseudo-distributed.ini.tmpl in sync. The new added content should be refleced same in both files.

Reply:
Changed the content in both conf.dist/hue.ini and conf/pseudo-distributed.ini.tmpl. They are in sync now.